hack/make-rules/test-cmd.sh

```diff
@@ -73,7 +74,8 @@ function run_kube_controller_manager() {
     --port="${CTLRMGR_PORT}" \
     --kube-api-content-type="${KUBE_TEST_API_TYPE-}" \
     --master="127.0.0.1:${API_PORT}" 1>&2 &
-  CTLRMGR_PID=$!
+  # Export CTLRMGR PID
+  export CTLRMGR_PID=$!
```

Comment:
Same here, for `run_kube_controller_manager`

Reply:
comment was added, this may? actually be read

Reply:
https://cs.k8s.io/?q=CTLRMGR_PID&i=nope&files=&repos=kubernetes/kubernetes